pages/infrastructure_providers-validators/how_to_set_up_full_node.md

```diff
@@ -1,70 +1,125 @@
-```
-2. You can tail the log to see the progress.
-```bash
-tail -f /tmp/fullnode.log
-```
-3. The full node is now syncing. To determine whether the full node is caught up with the chain head, please check the applicable block explorer to determine when it reaches the current block – an example block explorer is shown on https://www.mintscan.io/dydx
-
-## Things you can do with the full node
-GET CURRENT BLOCK: You can get the current block with this program https://github.com/chiwalfrm/dydxexamples/blob/1d46b7a75499205d9c1c1986ae4ae8f21b6c1385/v4block_subscribe.py
-
-Run it with the full node IP address and port `26657`:
-```bash
-python3 v4block_subscribe.py ws://<IPADDRESS>:26657
-```
-Where `<IPADDRESS>` is the IP address of your full node.
-![Full node usage example](../../artifacts/how_to_set_up_full_node_usage_example.png)
\ No newline at end of file
+# Set Up a Full Node
+
+> Examples on this page are for mainnet deployments by dYdX token holders. For information on alternative deployment types, including testnet deployments in the United States, see the [Network Constants page](../infrastructure_providers-network/network_constants.mdx).
+
+## Sytstem requirements
+To run a full node, the system that hosts the node must meet the following minimum requirements:
+
+- Linux (Ubuntu Server 22.04.3 or later recommended)
+- 8-core CPU (ARM or x86_64 architecture)
+- 64 GB RAM
+- 500 GB SSD NVMe Storage
+
+## Install `dydxprotocold`
+
+The `dydxprotocold` binary contains the software you need to operate a full node. You must use the same version of the software as the network to which you want to connect.
+
+1. Find the current version of the dYdX network.
+
+2. Find the matching `Release protocol` version from the [v4 Chain Releases](https://github.com/dydxprotocol/v4-chain/releases/) page. Download the compressed `dydxprotocold` file for your system.
+   
+   > For example, for protocol version 5.0.5 on an AMD system, download `dydxprotocold-v5.0.5-linux-amd64.tar.gz`.
+
+   Alternatively, you can install the latest release protocol by modifying [this curl command](https://gist.github.com/steinwaywhw/a4cd19cda655b8249d908261a62687f8) for your case.
+
+3. Extract the binary.
+
+   Extract the `.tar.gz` file that you downloaded. To run `dydxprotocold` from the command line, either rename the extracted file or create a symbolic link to the file:
+
+   **Option 1:** Rename the file. Edit the filename from `dydxprotocold-<version>-<architecture>` to simply `dydxprotocold`. You might need to add your current directory to your $PATH or move the file to a directory in your $PATH.
+
+   **Option 2:** Create a symbolic link to the file using the name `dydxprotocold`. Run the following command:
+   ```bash
+   ln -s /path/to/your/binary dydxprotocold
```

Comment:
Reformatting these to both use "/path/to/dydxprotocold-<version>-<architecture>" -- it's my opinion that variables/example names are clearer than "real" values, because it stands out from cases where we want them to run a command as-is and also helps users understand how things connect

Definitely debatable and we should apply the same formatting to all examples when we decide, so happy to discuss